lib/ansible/modules/cloud/amazon/aws_acm_facts.py

```diff
@@ -18,7 +18,7 @@
       - The domain name of an ACM certificate to limit the search to
     aliases:
       - name
-  status:
+  statuses:
```

Comment:
For backward compatibility and better UI experience, add 'status' as `aliases`.

Reply:
@Akasurde I don't see a `status` parameter in this module - only `statuses`.

Reply:
`status` option was available from the inception of this module, users might be using it. Changing directly to `statuses` will break existing playbooks. So I recommended using aliases instead of removing the parameter all together.

Reply:
@Akasurde you are incorrect. the option has always been statuses and the documentation has always been status.

Reply:
OK.